main.go

```diff
@@ -83,6 +83,7 @@ func main() {
 	flag.StringSliceVar(&ignoredUserGroups, "ignored-user-group", []string{}, "Names of the groups which requests must be ignored and proxy-passed to the upstream server")
 	flag.StringSliceVar(&ignoreImpersonationGroups, "ignored-impersonation-group", []string{}, "Names of the groups which are not used for impersonation (considered after impersonation-group-regexp)")
 	flag.StringVar(&impersonationGroupsRegexp, "impersonation-group-regexp", "", "Regular expression to match the groups which are considered for impersonation")
+	flag.BoolVar(&skipImpersonationReview, "skip-impersonation-review", false, "Skip verifiying the impersonation user and groups with subjectaccessreviews")
```

Comment:
Let's describe clearly the implications in disabling the impersonation review, with some all caps.